### PR TITLE
chore: elasticsearch-curator additonal indices

### PR DESCRIPTION
**Pull request by sebbrandt87**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

- added new indices to be deleted
- version 5.8.1
- defined FQDN for elasticsearch-kubeaddons-client

**Which issue(s) this PR fixes**:

no issue

**Special notes for your reviewer**:
5.8.1 is still compatible to ES 6 and 7
https://www.elastic.co/guide/en/elasticsearch/client/curator/current/version-compatibility.html

**Does this PR introduce a user-facing change?**:

```release-note
- Elasticsearch-Curator version 5.8.1
```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/elasticsearch-curator/5.7.x/elasticsearch-curator-3.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: elasticsearch-curator
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: elasticsearch-curator
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "5.8.1-1"
```
Comment (samvantran) [inline addons/elasticsearch-curator/5.7.x/elasticsearch-curator-3.yaml:10]: You have to change the directory. It's not 5.7 anymore

File: addons/elasticsearch-curator/5.8.x/elasticsearch-curator-1.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      configMaps:
+        # Delete indices older than 10 days
+        action_file_yml: |-
+          ---
+          actions:
+            1:
+              action: delete_indices
+              description: "Clean up ES by deleting indices older than 10 days"
+              options:
+                timeout_override:
```
Comment (samvantran) [inline addons/elasticsearch-curator/5.8.x/elasticsearch-curator-1.yaml:66]: do we need this key if there's no value?

Comment (sebbrandt87) [reply addons/elasticsearch-curator/5.8.x/elasticsearch-curator-1.yaml:66]: > Looks good with 1 question. Also please add a release-note about changing the default time for deleting indices down from 30 to 10 days.

I haven't changed that value, just updated the description. It was already 10days.
See https://github.com/mesosphere/kubernetes-base-addons/pull/374/commits/69311e2bf0279570829750ee0ffb2503e73e4f8c

Comment (samvantran) [reply addons/elasticsearch-curator/5.8.x/elasticsearch-curator-1.yaml:66]: oops you're right. nvm me.

**Pull request merged**